core/src/i18n/en.yaml

```
@@ -1056,7 +1056,12 @@ namespaces:
   create-modal:
     apply-limits: Apply limits per container
     apply-memory-quotas: Apply total memory quotas
+    container-limits: Container limits
+    create-limit-range: Create Limit Range
+    create-resource-quota: Create Resource Quota
     disable-sidecar: Disable sidecar injection
+    memory-limits: Memory limit
```

```suggestion
    memory-limits: Memory limits
```
Shouldn't we also use plural here?